taipy/_entrypoint.py

```diff
@@ -71,3 +76,58 @@ def _entrypoint():
 
     _TaipyParser._remove_argument("help")
     _TaipyParser._parser.print_help()
+
+
+import argparse
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Taipy server")
+    # REST server arguments
+    parser.add_argument("--rest-port", type=int, help="Port to run the REST server on")
+    parser.add_argument("--rest-host", type=str, help="Host to run the REST server on")
+    parser.add_argument("--rest-use-https", action="store_true", help="Use HTTPS for REST server")
+    parser.add_argument("--rest-ssl-cert", type=str, help="Path to SSL certificate for REST server")
+    parser.add_argument("--rest-ssl-key", type=str, help="Path to SSL key for REST server")
+    # GUI server arguments
+    parser.add_argument("--gui-port", type=int, help="Port to run the GUI server on")
+    parser.add_argument("--gui-host", type=str, help="Host to run the GUI server on")
+    parser.add_argument("--gui-use-https", action="store_true", help="Use HTTPS for GUI server")
+    parser.add_argument("--gui-ssl-cert", type=str, help="Path to SSL certificate for GUI server")
+    parser.add_argument("--gui-ssl-key", type=str, help="Path to SSL key for GUI server")
```

Comment:
The GUI arguments are already defined in the `_GuiCLI`. They should not be redefined here